docs/toc.yml

```diff
@@ -79,6 +81,8 @@
         href: develop/authorize-to-microsoft-graph-without-sso.md
       - name: Enable SSO in an add-in
         items:
+        - name: SSO quick start
```

Comment:
Since a Tip at the start of the quickstart recommends that they first read "Enable single sign-on", maybe that article should precede the quickstart in the TOC.

Reply:
Good suggestion -- I'll make this change.